plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/BackpressureRestClient.java

```diff
@@ -67,19 +72,43 @@ public BackpressureRestClient(RestClient delegate, ElasticsearchConfig config, T
 
     public void setHosts(HttpHost... hosts)
     {
-        delegate.setHosts(hosts);
+        delegate.setNodes(stream(hosts).map(Node::new).collect(toImmutableList()));
```

Comment:
Can we have something like this 
```
        delegate.setNodes(Stream.of(hosts)
                .map(Node::new)
                .collect(toImmutableList()));
```

Post `Stream` - we could move each operation to a new line